jetty-unixsocket/src/test/java/org/eclipse/jetty/unixsocket/UnixSocketTest.java

```diff
@@ -49,20 +47,18 @@
 
 public class UnixSocketTest
 {
-
     private Logger log = Log.getLogger( getClass() );
 
     Server server;
     HttpClient httpClient;
-    Path sockFile;
+    File sockFile;
 
     @Before
     public void before() throws Exception
     {
         server = null;
         httpClient = null;
-        sockFile = Files.createTempFile(new File("/tmp").toPath(), "unix", ".sock" );
-        Files.deleteIfExists(sockFile);
+        sockFile = File.createTempFile("unix", ".sock" );
```

Comment:
There should only be an `assumeFalse(OS.WINDOWS);` here in the `@Before`, all of the rest of the changes to UnixSocketTest are irrelevant.

Reply:
If you want to keep this, make sure the temp file is created within the `${maven.basedir}/target/` directory, that way you don't have to do any of the deleteIfExists logic to cleanup.

Use one of the MavenTestingUtils.getTarget*() methods that best suits your needs.